Framework/Crystal/src/OptimizeCrystalPlacement.cpp

```diff
@@ -420,33 +421,12 @@ void OptimizeCrystalPlacement::exec() {
   setProperty("FitInfoTable", RRes);
 
   //----------- update instrument -------------------------
-
-  IPeak &peak = Peaks->getPeak(0);
-  boost::shared_ptr<const Instrument> OldInstrument = peak.getInstrument();
-  boost::shared_ptr<const ParameterMap> pmap_old =
-      OldInstrument->getParameterMap();
-  auto pmap_new = boost::make_shared<ParameterMap>();
-
-  PeakHKLErrors::cLone(pmap_new, OldInstrument, pmap_old);
-
-  double L0 = peak.getL1();
   V3D newSampPos(Results["SampleXOffset"], Results["SampleYOffset"],
                  Results["SampleZOffset"]);
 
-  boost::shared_ptr<const Instrument> Inst = OldInstrument;
-
-  if (OldInstrument->isParametrized())
-    Inst = OldInstrument->baseInstrument();
-
-  auto NewInstrument = boost::make_shared<Geometry::Instrument>(Inst, pmap_new);
-
-  SCDCalibratePanels::FixUpSourceParameterMap(NewInstrument, L0, newSampPos,
-                                              pmap_old);
-
-  for (int i = 0; i < OutPeaks->getNumberPeaks(); i++)
-    OutPeaks->getPeak(i).setInstrument(NewInstrument);
```

Comment:
It is not clear to me either, it seems like it may have been to have access to pre-move and post-move information.

It seems unlikely that any of this would make sense if the peaks do not start with the same instrument. Is it worth adding this back to be safe? I can not see that it would make sense to change other peask to a different instrument, based on the first peak in the workspace.